core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -537,6 +536,169 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
     assert(mapStatus2(2).location.host === "hostB")
   }
 
+  test("All shuffle files on the executor should be cleaned up when executor lost " +
```

Comment:
For the bug fix PR, we had better add `SPARK-29551` prefix for the test case name.